src/bb/main.py

```
@@ -256,53 +151,28 @@ def scrape_banquet_prices() -> list[dict]:
+        vendors_with_pdfs = [v for v in vendors if v.get("price_lists")]
+        if vendors_with_pdfs:
+            print(f"\n📥 Downloading PDFs for {len(vendors_with_pdfs)} vendors...")
+
+            async with httpx.AsyncClient(headers=get_headers(), timeout=60) as client:
+                for idx, vendor in enumerate(vendors_with_pdfs, 1):
+                    print(f"\n[{idx}/{len(vendors_with_pdfs)}] {vendor['name']}")
+                    await download_pdfs_for_vendor(client, vendor)
+
         return vendors
 
     except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)